src/main/java/frc/util/GRTUtil.java

```diff
@@ -12,6 +12,7 @@ public static double clamp(double min, double x, double max) {
 		return Math.min(Math.max(min, x), max);
 	}
 
+	/** Takes the positive modulo */
 	public static double positiveMod(double x, double mod) {
```

Comment:
Why are we doing a floating point modulo but using the integer modulo operator `%` to realize it? This function is not working the way you think it is.